JM/CF/Scripts/1_Core/CommunityFramework/Files/CF_File.c

```diff
@@ -0,0 +1,167 @@
+class CF_File : Managed
+{
+	protected string m_FileName;
+	protected string m_Folder;
+	protected bool m_IsDirectory;
+	protected bool m_IsHidden;
+	protected bool m_IsReadOnly;
+	protected bool m_IsValid;
+
+	/**
+	 * @param pattern The search pattern
+	 * @param [inout] files An array where the list of files will be written to
+	 * @param flags If it should be looked in directories or .pak files, or both
+	 * 
+	 * @return If the pattern was valid
+	 */
+	static bool Find(string pattern, inout array<ref CF_File> files, FindFileFlags flags = 2)
```

Comment:
Can't we do `FindFileFlags flags = FindFileFlags.ALL`?

Reply:
Compile error unfortunately